src/routes/index.ts

```diff
@@ -47,5 +48,10 @@ const authRouter = Router();
 baseRouter.use("/auth", authRouter);
 authRouter.post("/login", adminLoginLimiter, AuthController.login);
 
+const apiKeyRouter = Router();
+baseRouter.use("/apiKey", apiKeyRouter);
```

Comment:
```suggestion
baseRouter.use("/api-key", apiKeyRouter);
```
Please don't use uppercase letters in url paths

Reply:
oh f...